plan/plan.go

```diff
@@ -40,11 +40,12 @@ type Config struct {
 	DryRun bool
 }
 
-func Load(ctx context.Context, cfg Config) (*Plan, error) {
+func Load(ctx context.Context, cfg Config, originalPlan string) (*Plan, error) {
 	ctx, span := otel.Tracer("dagger").Start(ctx, "plan.Load")
 	defer span.End()
 
-	log.Ctx(ctx).Debug().Interface("args", cfg.Args).Msg("loading plan")
+	log.Ctx(ctx).Debug().Interface("args", cfg.Args).Msg("loading plan with evaluated args")
+	log.Ctx(ctx).Debug().Interface("path", originalPlan).Msg("loading plan in path")
```

Comment:
Really good idea IMO but that is only accurate in that PR, you should remove it before merge

Reply:
Removed, thanks!